projects/core/src/error-handling/cx-error-handler.spec.ts

```diff
@@ -70,55 +61,6 @@ describe('CxErrorHandler', () => {
     ).and.callThrough();
   });
 
-  describe('error interceptors feature flag', () => {
```

Comment:
[MEDIUM]
Can we have some unit test that checks whether we chain the interceptors in the correct way?
1. e.g. provide 3 interceptors. Then pass an error to the chain. Then assert that the error was handled by A,B,C in the correct order?
2. e.g. provide 3 interceptors: A,B,C. Mock B so it doesn't pass an error to the _next_ interceptor. Then pass an error to the chain. Assert that the error was handled only by A and B, but not C.

Reply:
AD 1. test `should call all error interceptors` does exactly what you described. I've changed its desc to `should call all error interceptors in the correct order`
 
AD 2. I'll add such a test case.

Reply:
My bad. I missed that. Thanks!